packages/hepmcmerger/package.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class Hepmcmerger(Package):
+    """An EIC HepMC merger to combine signal and background events."""
+
+    # FIXME: Add a proper url for your package's homepage here.
+    homepage = "https://github.com/eic/HEPMC_Merger"
+    url = "https://github.com/eic/HEPMC_Merger/releases/tag/v1.0.4"
```

Comment:
This needs to be a tar.gz file. Your downloading a github webpage.

Reply:
Thanks for the catch